Library/Formula/bmdtools.rb

```diff
@@ -0,0 +1,21 @@
+require 'formula'
+ 
+class Bmdtools < Formula
+  homepage 'http://github.com/lu-zero/bmdtools'
+  url 'https://github.com/lu-zero/bmdtools/archive/v0.1.zip'
+  sha256 'cc2a2f9c91b019dadea6dff591b07ceb7db05ce8c25ff405f715206ee81c7664'
+  head 'https://github.com/lu-zero/bmdtools.git'
+ 
+  env :std
```

Comment:
Tested. sdtenv isn't needed, removed in e9fc1357407029d3454519968a523374d32e918f